docs/design/generics/details.md

```diff
@@ -3814,55 +3869,39 @@ equal.
 ```
 interface Foo(T:! type) { ... }
 class Pair(T:! type, U:! type) { ... }
-external impl forall [T:! type] Pair(T, T) as Foo(T) { ... }
+impl forall [T:! type] Pair(T, T) as Foo(T) { ... }
 ```
 
-You may also define the `impl` inline, in which case it can be internal:
+You may also define the `impl` inline, but it may not be combined with the
+`extend` keyword:
```

Comment:
Could 3847-3856 and 3875-3882 be unified, since they are both explaining that an `impl` can be inline without `extend`?

Reply:
I've added the words "as before" to make it clear this is just another example of the same thing.

Reply:
Just to note, it still feels redundant to me as both examples seem like they're demonstrating an absence of `extend`, but maybe others will find the repetition useful.